src/services/car/__tests__/messageLead.test.ts

```diff
@@ -364,6 +367,80 @@ describe("Car API", () => {
     })
   })
 
+  describe("#fetchDealerWhatsappLeads", () => {
+    const { content, pagination } = PaginatedLeads([
+      SearchWhatsappLeadFactory(),
+    ])
+
+    beforeEach(() => {
+      fetchMock.mockClear()
+      fetchMock.mockResponse(
+        JSON.stringify({
+          content: content,
+          ...pagination,
+        })
+      )
+    })
+
+    it("calls correct endpoint", async () => {
+      await fetchDealerWhatsappLeads({
+        dealerId: 1234,
+        query: {
+          page: 2,
+          size: 7,
+          sort: defaultLeadSort,
+        },
+        options: {
+          accessToken: "DummyTokenString",
+        },
+      })
+
+      expect(fetch).toHaveBeenCalledWith(
+        expect.stringMatching(
+          /\/dealers\/1234\/whats-app-tracking-entries\?page=1&size=7&sort=createdDate%2Cdesc$/
+        ),
+        expect.any(Object)
+      )
+    })
+
+    it("should trow error if accessToken is not passed", async () => {
+      let error
+      try {
+        await fetchDealerCallLeads({
```

Comment:
Heads up, you're calling another method that you promised testing. You probably want
```suggestion
        await fetchDealerWhatsappLeads({
```